test/k6/src/setup.js

```diff
@@ -0,0 +1,88 @@
+import http from "k6/http";
+import { check } from "k6";
+import * as tokenGenerator from "./api/token-generator.js";
+import * as config from "../config.js";
+import { stopIterationOnFail } from "../errorhandler.js";
+
+const environment = __ENV.env.toLowerCase();
+
+/*
+ * generate an altinn token for TTD based on the environment using AltinnTestTools
+ * @returns altinn token with the provided scopes for an org/appowner
+ */
+export function getAltinnTokenForTTD(scopes) {
```

Comment:
It will be great to extract these functions to a library at some point soon. But not now 😸